gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Issue - Fix undefined `flattened_list?` and `flattened_map?` methods in RPCv2 parser.
```

Comment:
I don't think this changelog entry is useful to know in customer POV.

I would do:
> Issue - For `awsQueryCompatible` services, RPCv2 parser handles flattened list and map members correctly.